release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -941,9 +952,7 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-4-11-builds-bug-fixes"]
 ==== Builds
 
-[discrete]
-[id="ocp-4-11-jenkins-bug-fixes"]
-==== Jenkins
+* Before this update, Buildah printed steps to the log file by default. As a result, the contents of environment variables could be exposed by the default build settings. This update resolves the issue. The `--quiet` option is appended to Buildah builds for log levels below 2. Now Buildah build steps are no longer printed to the log when a build level below 2 is specified, or no build log level is specified. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1996883[BZ#1996883])
```

Comment:
This is not correct. That bugzilla got backed out, the new way is to set the "BUILDAH_QUIET" env var on the buildconfig

Reply:
Replaced this RN with the one for https://bugzilla.redhat.com/show_bug.cgi?id=2099991. Thanks.